examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,748 @@
+{
```

Comment:
<p>I'm positive you can use <code>az.plot_ppc</code> with the <code>kind</code> and <code>coords</code> kwargs here (see https://arviz-devs.github.io/arviz/api/generated/arviz.plot_ppc.html).</p><p>The nice is that you'll use <code>xarray</code> indexing instead of classic numpy indexing. <code>i</code> and <code>j</code> are the sample and category, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I can get `plot_ppc` to plot something using the `coords=` kwarg but it's not at all clear to me that it's set up well to produce a histogram like the one I made manually.

Here's what I tried:
```python
az.plot_ppc(
    trace_marginalized,
    kind='scatter',
    var_names=['obs'],
    coords=dict(obs_dim_0=[9], obs_dim_1=[2])
)
```

And here's the result:
![image](https://user-images.githubusercontent.com/711411/104794377-1b350780-575c-11eb-943c-717bab6bbd42.png)

Having not read up on PPCs in ArviZ, it's not super apparent to me what that plot's trying to show. What's the y-axis? Frequency I assume, but then what are the `'o'`-styled points? In any case, I'd much prefer to use a bar-chart style histogram than a KDE-plot, given that the values of `obs` are discrete integers.

Anyway, that's all to say that I don't think `plot_ppc` is currently the correct tool for this job.

Instead, to motivate the use of a PPC, I think I'll run a _third_ model that's just a plain Multinomial distribution, and use the PPC to demonstrate that it does not fit the over-dispersed data well.

Reply:
Interesting 🤔 Let me take a look ASAP, because I'm still not convinced that it's not currently possible with `plot_ppc`. Did you allow push by maintainers on this PR?
Also tagging @OriolAbril and @aloctavodia, who are ArviZ wizards 🧙‍♂️ 

>  I'll run a third model that's just a plain Multinomial distribution, and use the PPC to demonstrate that it does not fit the over-dispersed data well.

Great idea!

Reply:
> I'm still not convinced that it's not currently possible with plot_ppc.

Looking forward to being proven wrong 😄

> Did you allow push by maintainers on this PR?

Looks like that box is checked, yes.

Reply:
Perfect, thanks. I'll tell you when I got some time to look over it

> Looking forward to being proven wrong 😄

Ha ha, sorry, that's not what I meant -- rather, I'm seeing this as a nice, non trivial test case for `plot_ppc`'s capacities 😉